src/routes/+layout.svelte

```diff
@@ -18,16 +18,17 @@
     import { navbar_middle_section_variant } from "$store/navbar";
     import { theme } from "$store/theme";
     // Most of your app wide CSS should be put in this file
-    import "../app.css";
+    import "../app.postcss";
```

Comment:
![Screenshot from 2023-08-16 05-31-10](https://github.com/tokitou-san/CoreProject-V3-UI/assets/114811070/5a30d8ac-e0fa-454d-999a-8afd5a565cf3)

Unfortunately custom fonts are not working.

Reply:
Hi @tokitou-san! This is an issue with `@skeletonlabs/tw-plugin` version `0.0.1-rc.0`. Updating to `0.0.1-rc.3` should fix it!

Reply:
> Hi @tokitou-san! This is an issue with @skeletonlabs/tw-plugin version 0.0.1-rc.0. Updating to 0.0.1-rc.3 should fix 

Hi @AdrianGonz97, issue fixed.
Thanks a lot :)